src/components/Form/Form.css

```diff
@@ -57,17 +57,18 @@
   height: 100%;
   top: 0;
   left: 0;
-  z-index: 1;
   background-color: rgba(0, 0, 0, .5); 
   visibility: hidden;
   opacity:0;
-  transition:visibility 0s linear .5s, opacity .5s linear;
+  z-index: 1000;
+  transition: visibility 0s linear .5s, opacity .5s linear, z-index .5s linear;
```

Comment:
i haven't seen what this looks like live, but very curious! i'll try to pull locally because i'm guessing this may be something worth all chatting through design approach on BUT i dont think it needs to block merging first version :)